app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -460,18 +460,18 @@ export function postRNWebviewMessage(prop, value) {
 }
 
 const CODE_BLOCKS_REGEX =
-  /^(    |\t).*|`[^`]+`|^```[^]*?^```|\[code\][^]*?\[\/code\]/gm;
-//                        |      ^     |   ^   |      ^      |           ^           |
-//                               |         |          |                  |
-//                               |         |          |       code blocks between [code]
-//                               |         |          |
-//                               |         |          +--- code blocks between three backticks
-//                               |         |
-//                               |         +----- inline code between backticks
-//                               |
-//                               +------- paragraphs starting with 4 spaces or tab
-
-const OPEN_CODE_BLOCKS_REGEX = /`[^`]+|^```[^]*?|\[code\][^]*?/gm;
+  /^(  |\t).*|`[^`]+`|^```[^]*?^```|\[code\][^]*?\[\/code\]/gm;
+//|      ^     |   ^   |      ^      |           ^           |
+//       |         |          |                  |
+//       |         |          |       code blocks between [code]
+//       |         |          |
+//       |         |          +--- code blocks between three backticks
+//       |         |
+//       |         +----- inline code between backticks
+//       |
+//       +------- paragraphs starting with 2 spaces or tab
```

Comment:
Thanks for addressing the comment alignment, but this looks misaligned still, the pipe (|) character in the first line should align with the first pipe character in the regex, I believe. (And for the other rows, the pipe characters should line up vertically.)

Reply:
Ah oops, yeah I think it got misaligned again when I changed to 2 spaces instead of 4. I think I should have it correct now?